frontend/src/components/AdminDash/AppInfo.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { Component } from 'react';
+import { Container, Col, Row, Progress, Label, Button } from 'reactstrap'
+import styles from '../../styles/AppInfo.module.css';
+
+const userData = [
+  {
+    role: 'volunteer',
+    bio: {
+      first_name: 'John',
+      last_name: 'Adams',
+      phone_number: '(888)-888-8888',
+      email: 'jadams@foundingfathers.gov',
+      date_of_birth: 'October 31, 1735',
+      street_address: '350 Ferst Drive',
+      city: 'Atlanta',
+      state: 'Georgia',
+      zip_code: '30332'
+    },
+    history: {
+      volunteer_interest_cause: 'Well, I just helped found a nation and finally have some time to give back to my community. This cause really caught my eye!',
+      volunteer_support: 'I value the pursuit of happiness and collaborating with other brothers and sisters of our great nation.',
+      volunteer_commitment: 'Due to my obligations to restructuring the government, I must be honest and let you know that I cannot be 100% committed.',
+      previous_volunteer_experience: 'Not sure if you would call it volunteer work, but there was certainly a lot of work with no pay'
+    },
+    availability: {
+      weekday_mornings: true,
+      weekday_afternoons: true,
+      weekday_evenings: false,
+      weekend_mornings: false,
+      weekend_afternoons: true,
+      weekend_evenings: false
+    }
+  }
+];
+
+export default class AppInfo extends Component {
```

Comment:
Seems like this component should maybe be a functional presentational component, as opposed to being a class

Reply:
Also should `userData` be passed in as a prop into `AppInfo`?

Reply:
Also, seems like we're calling this same component `AppInfo` and `ApplicantInfo` interchangeably in our code base. Might suggest we pick a name and stick with it. Personally I like `ApplicantInfo` :)